bin/benchpark

```diff
@@ -163,7 +215,21 @@ def benchpark_setup(subparsers, actions_dict):
         help="Where to install packages and store results for the experiments. Benchpark expects to manage this directory, and it should be empty/nonexistent the first time you run benchpark setup experiments.",
     )
 
+    system_create_parser = subparsers.add_parser(
+        "system-create", help="Initialize a system config"
+    )
+    system_create_parser.add_argument(
+        "--template",
+        action="append",
+        help="Choose common config settings in templates/",
+    )
+    system_create_parser.add_argument(
+        "--from", dest="use_existing", type=str, help="Copy an existing system config"
```

Comment:
Nice! We'd talked about this prior but I really like the idea of being able to copy from an existing system and just tweak it as necessary.